scripts/travis.sh

```diff
@@ -5,8 +5,11 @@ SUBFOLDER=$1
 
 function test {
     set -ev
+    npm install
+    npm run build
```

Comment:
Shouldn't this be included in `docs/package.json`? Guess in the build step. Because either way we can't build docs without running `npm install` on the parent directory. Also, this forces two builds when we're just testing the library and not the docs.

Reply:
its apparently not doing for parent folder from the child folder